chart/values.yaml

```diff
@@ -0,0 +1,59 @@
+# -- Number of replicas for the deployment
+replicaCount: 1
+
+common:
+  # -- Labels to add to all deployed resources
+  labels: {}
+  # -- Namespace to deploy the resources
+  namespace: "yunikorn"
+
+image:
+  # -- Docker registry
+  registry: "docker.io"
+  # -- Docker image repository
+  repository: "gresearch/yunikorn-history-server"
+  # -- Docker image tag
+  tag: "latest"
+  # -- Docker image pull policy
+  pullPolicy: "IfNotPresent"
+
+service:
+  # -- Service type
+  type: LoadBalancer
+  # -- Service port
+  port: 8989
+  # -- Service target port
+  targetPort: 8989
+
+config:
+  # -- Yunikorn scheduler host
+  yunikornHost: ""
+  # -- Yunikorn scheduler port
+  yunikornPort: ""
+  # -- Yunikorn scheduler protocol
+  yunikornProtocol: "http"
+  # -- YHS database host
+  dbHost: ""
```

Comment:
this can be groupd in a `db` block:
```
db:
  host:
  port:
```

Reply:
Yeah, initially I had done that but after reading [this](https://helm.sh/docs/chart_best_practices/values/#flat-or-nested-values) helm best practice guide I thought I should flatten as much as possible. I can group them again if grouping is better.

Reply:
Updated the structure of `values.yaml`